src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -235,6 +235,24 @@
      * modifiers, type, the identifier name, and an optional
      * assignment statement.
      *
+     * <p>For example:</p>
+     * <pre>
+     * private int PI = 3.14;
```

Comment:
it seems like `private` modifier is redundant here, what do you think if we remove it?

Reply:
I have added `private` to show access modifier node in the AST, I think replacing it with final seems like a better idea.